nanome_rmsd/RMSD.py

```diff
@@ -348,14 +348,23 @@ def on_select_received(self, workspace):
                 percentage_count += 1/(total_percentage*2)
                 self._menu.change_loading_percentage(percentage_count)
 
-            for x in self._mobile[:-1]:
+            for x in self._mobile:
                 selection.global_align(x , self._target) 
                 percentage_count += 1/(total_percentage*2)
                 self._menu.change_loading_percentage(percentage_count )
-        
-        
-        if (self.args.select.lower() == "local"):
-            selection.local_align(self._mobile,self._target)
+
+        elif (self.args.select.lower() == "local" and self.args.align_sequence == True):
```

Comment:
Why are you using a string instead of a boolean or a enum?

Reply:
not using a boolean because there might be a third kind of alignment "glocal" alignment. Enum not sure, I've never really used it in python but I can look into it :)